controllers/result_test.go

```diff
@@ -21,6 +21,22 @@ var _ = Describe("ResultController", func() {
 
 				Expect(response.StatusCode).To(Equal(http.StatusOK))
 			})
+
+			Context("given user have results", func() {
```

Comment:
```suggestion
			Context("given the user have results", func() {
```

Reply:
✅  updated in [466be1a](https://github.com/carryall/go-google-scraper-challenge/pull/65/commits/466be1ac5573f03b4aac96aaa7d391379fb53be0)